src/OpenIdConnect.php

```diff
@@ -346,9 +422,9 @@ protected function createToken(array $tokenConfig = [])
     protected function loadJws($jws)
     {
         try {
-            $jwkSet = JWKFactory::createFromJKU($this->getConfigParam('jwks_uri'));
-            $loader = new Loader();
-            return $loader->loadAndVerifySignatureUsingKeySet($jws, $jwkSet, $this->allowedJwsAlgorithms)->getPayload();
+            $jwsLoader = $this->getJwsLoader();
+            $jwsVerified = $jwsLoader->loadAndVerifyWithKeySet($jws, $this->getJwkSet(), $signature);
```

Comment:
`$signature` is not initialized.

Reply:
They use it has a return variable that the reason I didn't init it.

```
/**
     * This method will try to load and verify the token using the given key set.
     * It returns a JWS and will populate the $signature variable in case of success, otherwise an exception is thrown.
     *
     * @throws Exception if the token cannot be loaded or verified
     */
    public function loadAndVerifyWithKeySet(string $token, JWKSet $keyset, ?int &$signature, ?string $payload = null): JWS
```
I cant set it to 0 or -1 and it will be override afterward, would it be ok ?